libnmstate/nm/ovs.py

```diff
@@ -175,10 +185,17 @@ def _get_bridge_port_info(port_profile, devices_info):
                 port_name, port_setting, port_slave_names
             )
             port_info.update(port_lag_info)
-        if vlan_mode:
-            port_info["vlan-mode"] = vlan_mode
-            port_info["access-tag"] = port_setting.props.tag
 
+        if vlan_mode:
+            nmstate_vlan_mode = NM_OVS_VLAN_MODE_MAP.get(vlan_mode)
+            if nmstate_vlan_mode == OB.Port.Vlan.Mode.UNKNOWN:
+                logging.warning(
+                    f"OVS Port VLAN mode '{vlan_mode}' is not supported yet"
+                )
+            port_info[OB.Port.VLAN_SUBTREE] = {
+                OB.Port.Vlan.MODE: nmstate_vlan_mode,
+                OB.Port.Vlan.TAG: port_setting.get_tag(),
```

Comment:
`port_setting.get_tag()` can return None, no? Maybe add an `or 0`?

Reply:
The document and NetworkManager confirmed, it will return a integer and 0 for missing.